src/OneExplorer/ConfigObject.ts

```diff
@@ -321,26 +334,26 @@ export class ConfigObj {
     if (artifacts.length > 1) {
       // TODO Notify the error with a better UX
       // EX. put question mark next to the config icon
-      Logger.warn('OneExplorer', `There are multiple input models in the configuration(${uri}).`);
+      Logger.warn(
+          'OneExplorer', `There are multiple input models in the configuration(${filePath}).`);
```

Comment:
(optional)
What about defining the log tag of `OneExplorer` keyword?

Reply:
What do you mean..? Can you explain more?

Reply:
I mean you can make a logtag for `OneExplorer`.

```
const logtag = 'OneExplorer';
Logger.warn(logtag, ...);
```

Reply:
Oh, sure... Can I apply the change later then?

Reply:
Yes. It's up to you to apply this suggestion. :)